english/dance/bake-a-cake/bake-a-cake.md

```diff
@@ -1,6 +1,10 @@
 You've gotten hungry from all the dancing, so you decided to bake a cake.
 
-You tasted the cake. Yummy!
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.